Mathlib/Algebra/Ring/CentroidHom.lean

```diff
@@ -399,6 +407,57 @@ theorem comp_mul_comm (T S : CentroidHom α) (a b : α) : (T ∘ S) (a * b) = (S
   rw [map_mul_right, map_mul_left, ← map_mul_right, ← map_mul_left]
 #align centroid_hom.comp_mul_comm CentroidHom.comp_mul_comm
 
+local notation "L" => AddMonoid.End.mulLeft
+local notation "R" => AddMonoid.End.mulRight
+
+/-- Ring hom from the centroid into the additive monoid endomorphisms. -/
+def toEndRingHom (α : Type*) [NonUnitalNonAssocSemiring α] :
+    RingHom (CentroidHom α) (AddMonoid.End α) where
+  toFun := toEnd
+  map_add' := toEnd_add
+  map_zero' := toEnd_zero
+  map_one' := toEnd_one
+  map_mul' := toEnd_mul
+
+lemma toEnd'_apply (f : CentroidHom α) : toEndRingHom α f = f.toEnd := rfl
```

Comment:
why the prime? why not `toEnd_apply`? (or better: `toRingHomEnd_apply`.) And probably you can just get this with `@[simps]` on `toRingHomEnd`.

Reply:
We now get this from:
```
@[simps]
def toEndRingHom : CentroidHom α →+* AddMonoid.End α where
```